core/chains/evm/config/v2/config.go

```diff
@@ -427,7 +427,7 @@ type GasEstimator struct {
 	LimitMax        *uint32
 	LimitMultiplier *decimal.Decimal
 	LimitTransfer   *uint32
-	LimitJobType    *GasLimitJobType
+	LimitJobType    GasLimitJobType `toml:",omitempty"`
```

Comment:
This struct is a bit of a special case, holding only nilable fields w/o defaults, but I think we'll be able to use this non-pointer/omitempty simplification in other places.